packages/patternfly-4/react-core/src/components/Tabs/examples/TabChildren.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Tabs, Tab, TabChild } from '@patternfly/react-core';
+
+class TabChildren extends React.Component {
+  state = {
+    activeKey: 0,
+  };
+
+  // Toggle currently active tab
+  handleTabClick = (event, tabIndex) => {
+    this.setState({
+      activeTabKey: tabIndex
+    });
+  };
+
+  render() {
+    return <React.Fragment>
+      <Tabs activeKey={this.state.activeTabKey} onSelect={this.handleTabClick}>
+        <Tab eventKey={0} title="Tab item 1" tabChild="tab1Section"></Tab>
+        <Tab eventKey={1} title="Tab item 2" tabChild="tab2Section"></Tab>
+        <Tab eventKey={2} title="Tab item 3" tabChild="tab3Section"></Tab>
+      </Tabs>
+      <div>
+        <TabChild eventKey={0} id="tab1Section">Tab 1 section</TabChild>
+        <TabChild eventKey={1} id="tab2Section">Tab 2 section</TabChild>
+        <TabChild eventKey={2} id="tab3Section">Tab 3 section</TabChild>
+      </div>
+    </React.Fragment>;
+  }
```

Comment:
Thinking this should be done by refs so it's more React-idiomatic. 

The link below describes a re-usability issue when using IDs / getElementById.

https://www.javascriptstuff.com/use-refs-not-ids/

"When you use refs, you get the node rendered by `this` component, and you can re-use your component as much as you want."

Reply:
Yeah, I had it working with refs, but I was getting a warning in the console from whatever builds the docs. As long as the warning's okay, I can add that back in.

Reply:
Depends on the warnings. You may need to add a `key` property if you're iterating with a map?

Reply:
`Warning: Failed prop type: Invalid prop components.TabChild of type object supplied to ComponentDocs, expected function.`

It think it's due to using forwarding refs: https://reactjs.org/docs/forwarding-refs.html

Reply:
That doesn't sound related to using refs Vs IDs, but something about how the `Tabs.docs.js` file is processed. I suspect this is a Type issue? Perhaps related to the use of `FunctionComponent` in `TabChild.d.ts`?

Reply:
Perhaps something do to with how `TabChildProps` extends `HTMLProps`? looks different than `TabsProps` in `tabs.d.ts`